### ISSUE TITLE
RIGHT CLICK DONT WORK

### ISSUE THREAD

**Issue by pablo2030** [ChrisRM/material-theme-jetbrains#1082]
Hello I use phpstorm but the right click does not work phpstorm 2018.3 and 2018.3.1 even my intellij idea 2018.3, and when I deactivate Material Theme UI everything works...

**Comment by mallowigi** [comment]
Can you provide more information about your system?

**Comment by pablo2030** [comment]
I am under macOS 10.13.6 when I run phpstrom when I click on project to work everything but on this or any project it imports when I click right for the menu nothing is displayed to create a new folder or a work file
example laravel when I click right on public folder for a folder img I do not have the menu to create it and even its block I can not write anything I must leave the phpstorm but right now I must go in root of my work record to create for folder or file manually and it's very punitive I have to have a bug parch when I desative material-theme-ui everything works

**Comment by mallowigi** [comment]
Can you provide information about your plugins?

**Comment by alex-mohemian** [comment]
@mallowigi thank you for your quick reply and sorry for not opening a seperate ticket for this. Let me know if you want a new ticket. Unfortunately the bug isn't fixed in 3.20 -- in fact that was the version I had installed when I took the screenshot above!

![screenshot 2018-12-21 at 13 07 43](https://user-images.githubusercontent.com/34009288/50342033-867e6200-0521-11e9-9998-95919dd1aeda.png)

**Comment by mallowigi** [comment]
That concerns me, perhaps this is another issue? Anyway, can you check if the option “Material Design components” is unchecked and if yes, please check it and restart. 

If the problem solves itself, try unchecking it again, restart and tell me if the problem reproduces because it shouldn’t

**Comment by alex-mohemian** [comment]
> That concerns me, perhaps this is another issue? Anyway, can you check if the option “Material Design components” is unchecked and if yes, please check it and restart.

Interesting! Yes it was unchecked (although I don't know why). I checked it and restarted -- that fixed it, like you said.

I unchecked it again and restarted, and the problem _comes back_!

If Material Design Components is checked, the right click issue is also fixed. With the option unchecked, right clicking is not possible. No other extensions enabled. Now on 2018.3.2.
